src/main/java/no/nav/tag/kontaktskjema/DataSourceConfiguration.java

```diff
@@ -21,7 +22,6 @@
     @Bean
     public DataSource dataSource() {
         DriverManagerDataSource dataSource = new DriverManagerDataSource();
-        dataSource.setDriverClassName("org.postgresql.Driver");
```

Comment:
Er rimelig sikker på at vi ikke trenger å sette driver, men at datasourcen selv klarer å utlede det ut fra db-url. Det funker iallfall for h2 og Oracle, så antar at det også funker for postgres. Dermed kan denne Beanen brukes til flere databasetyper, ikke bare postgres.